rock-paper-scissors.js

```diff
@@ -0,0 +1,44 @@
+// 7. In this kata, your task is to implement an extended version of the famous rock-paper-scissors game.
+
+function willRockWin(opponent) {
+    return opponent === 'scissors' || opponent === 'lizard';
+}
+
+function willScissorsWin(opponent) {
+    return opponent === 'paper' || opponent === 'lizard';
+}
+
+function willLizardWin(opponent) {
+    return opponent === 'paper' || opponent === 'spock';
+}
+
+function willPaperWin(opponent) {
+    return opponent === 'spock' || opponent === 'rock';
+}
+
+function willSpockWin(opponent) {
+    return opponent === 'scissors' || opponent === 'rock';
+}
+
+function extendedVersionOfRockPaperScissors(playerOne, playerTwo){
+    const rockWin = willRockWin(playerTwo);
+    const scissorsWin = willScissorsWin(playerTwo);
```

Comment:
Do we really need to call the `willScissorsWin` function if the `playerOne` is a rock? Restructure this function a bit, taking it into account.